packages/ui/src/components/Button.svelte

```diff
@@ -85,6 +86,7 @@
 <button
   use:tooltip={showTooltip}
   bind:this={input}
+  bind:clientWidth={realWidth}
```

Comment:
I'm dislike adding new properly to Button, please have Labels control to fit into available size and show One/Two labels in case they fit or compact form in case it is required. Please calculate based on labels count of symbols and possible symbol font fit.